LEGO1/lego/legoomni/src/worlds/registrationbook.cpp

```diff
@@ -440,7 +454,18 @@ void RegistrationBook::ReadyWorld()
 		}
 	}
 
-	if (m_infocenterState->m_letters[0] != NULL) {
+	// TODO: Remove this one if the function mismatches again
```

Comment:
No need to add something like this. We have a lot of functions susceptible to entropy so unless this is a permanent fix that keeps it at 100% at all circumstances there's not much point to it

Reply:
I went back and validated that this is actually due to entropy. Removing the void call and adding
```
class Stuff {
	void Tmp();
};

void Stuff::Tmp() {

}
```
above the function makes it go back up to 100 % (non-effective).